evalml/automl/automl_algorithm/default_algorithm.py

```diff
@@ -427,17 +430,18 @@ def _transform_parameters(self, pipeline, proposed_parameters):
 
     def _make_split_pipeline(self, estimator, pipeline_name=None):
         numeric_pipeline_parameters = {
-            "Select Columns Transformer": {"columns": self._selected_cols}
+            "Select Columns Transformer": {"columns": self._selected_cols},
+            "Select Columns By Type Transformer": {"column_types": ["numeric"]},
         }
         numeric_pipeline = make_pipeline(
-            self.X,
+            self.X.ww.select(exclude="category"),
```

Comment:
here I exclude all `category` semantic tag columns so there won't be preprocessing components for those components.